linux.version

```diff
@@ -1,11 +1,11 @@
 {
   "linux": {
-    "version": "2.32.2.20240425",
+    "version": "2.32.2.20240516",
     "latest": "true",
     "build": "1",
     "fluent-bit": "1.9.10",
     "kinesis-plugin": "v1.10.1",
     "firehose-plugin": "v1.7.1",
     "cloudwatch-plugin": "v1.9.4"
   }
-}
\ No newline at end of file
+}
```

Comment:
Would you remove the extra indentation here? It's not the best format, but let's not make unnecessary changes in the versioning file.